extensions/amp-facebook/1.0/README.md

```diff
@@ -0,0 +1,438 @@
+---
+$category@: layout
+formats:
+  - websites
+teaser:
+  text: Displays a Facebook post, video, or comment.
+---
+
+# Bento Facebook
+
+## Usage
+
+The Bento Facebook component allows you to embed a Facebook post, a Facebook video, or a comment on a Facebook post. It can be used as a web component [`<bento-facebook>`](#web-component), or as a Preact/React functional component [`<BentoFacebook>`](#preact/react-component). 
+
+### Web Component
+
+you must include each Bento component's required CSS library before adding custom styles in order to guarantee proper loading. Or use the lightweight pre-uprgrade styles available inline. See [Layout and Style](#layout-and-style).
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-facebook
+```
+
+```javascript
+import '@ampproject/bento-facebook';
+```
+
+[/example]
+
+#### Example: Include via `<script>`
+
+[example preview="top-frame" playground="false"]
+
+```html
+<head>
+  <script src="https://cdn.ampproject.org/custom-elements-polyfill.js"></script>
+  <!-- These styles prevent Cumulative Layout Shift on the unupgraded custom element -->
+  <style data-bento-boilerplate>
+    bento-facebook {
+      display: block;
+      overflow: hidden;
+      position: relative;
+    }
+  </style>
+  <!-- TODO(wg-bento): Once available, change src to bento-facebook.js -->
+  <script async src="https://cdn.ampproject.org/v0/amp-facebook-1.0.js"></script>
+  <style>
+    bento-facebook{
+      width: 375px;
+      height: 472px;
+    }
+  </style>
+</head>
+
+<!-- Facebook Post -->
+<amp-facebook id="facebook-post" width="552" height="310"
+    layout="responsive"
+    data-href="https://www.facebook.com/ParksCanada/posts/1712989015384373">
+</amp-facebook>
```

Comment:
Please replace all `amp-facebook` with `bento-facebook` and remove all `width`/`height`/`layout` attrs
```suggestion
<bento-facebook id="facebook-post" 
    data-href="https://www.facebook.com/ParksCanada/posts/1712989015384373">
</bento-facebook>
```